src/modules/navigation/components/Navigation/__mocks__/navigation.tsx

```diff
@@ -145,26 +145,26 @@ export const MOCK_ITEMS_LEFT: NavigationItem[] = [
 				styles['c-navigation__link--dropdown'],
 			]),
 		}),
-		id: 'beheer',
+		id: 'nav__beheer',
 		hasDivider: 'md',
 		children: [
 			{
-				node: renderLink('Aanvragen', '#', {
+				node: renderLink('Aanvragen', 'beheer/aanvragen', {
```

Comment:
Q: Should these be translated as well?
They are only mocks, but they might be used as the basis for the actual navigation.

Reply:
From what I recall this project will have a similar setup to AvO where navigation items will be db-driven. Making these mock items translatable will only cause confusion in the long run imo.

Reply:
These should have a / at the start though, otherwise you can link to this url from the leeszaal page:
```
http://localhost:3200/leeszaal/beheer/aanvragen
```